BadgeFramework/audio_parser_V0.py

```diff
@@ -15,12 +26,11 @@ def main(fn):
 
             path_wav_output = path_raw_input.parent / (path_raw_input.stem + ".wav")
 
+            buffer_dtype = np.int16 # 16-bit PCM
             if path_raw_input.stem[4:6] == "LO":
                 sample_rate = LOW_SAMPLE_RATE  # Low frequency sampling
-                buffer_dtype = np.int16 # 16-bit PCM
             elif path_raw_input.stem[4:6] == "HI":
                 sample_rate = HIGH_SAMPLE_RATE  # High frequency sampling
-                buffer_dtype = np.int32 # 32-bit PCM
```

Comment:
Do you have an idea why in the old samples where at 32 bit? As I understand it's just about quality of the audio, isn't it?

Reply:
to note: the decoding script might have worked because it assumed 32 bit PCM but assumed sample rate was half the actual sample rate (8K instead of 16K)

Reply:
OMG, this explains a lot actually. Then I'm happy with this change.